src/applications/gi-sandbox/containers/ProfilePageHeader.jsx

```diff
@@ -0,0 +1,323 @@
+          Contact Information
+        </IconWithInfo>
+      </a>
+    </div>
+  );
+  const renderProfilePageCard = () => (
+    <div className="usa-width-three-fourths">
+      <div className="vads-u-background-color--gray-lightest profile-card">
+        {schoolClassification}
+        <div className="vads-u-padding-left--2">
+          <h1 tabIndex={-1} className={titleClasses}>
+            {it.name}
+          </h1>
+          <p>{formattedAddress}</p>
+          {displayStars && (
+            <div className={starClasses}>
+              <span className="vads-u-font-size--sm">
+                <RatingsStars rating={it.ratingAverage} />
+              </span>{' '}
+              <span className="vads-u-padding-left--1 vads-u-padding-right--1">
+                |
+              </span>{' '}
+              <span className="vads-u-font-weight--bold vads-u-padding-right--1">
+                {stars.display} of 5
+              </span>{' '}
+              (
+              <a
+                href="#profile-school-ratings"
+                onClick={() => recordEvent({ event: 'nav-jumplink-click' })}
+              >
+                See {it.ratingCount} ratings by Veterans
+              </a>
+              )
+            </div>
+          )}
+          {it.studentCount > 0 && (
+            <p>
+              <strong>{formatNumber(it.studentCount)}</strong> GI Bill students
+              (
+              <button
+                type="button"
+                className="va-button-link learn-more-button"
+                onClick={onGiBillLearnMore}
+                aria-label={ariaLabels.learnMore.numberOfStudents}
+              >
+                Learn more
+              </button>
+              )
+            </p>
+          )}
+        </div>
+        {it.cautionFlags.length > 0 && (
+          <div className="caution-flag-section">
+            <CautionFlagAdditionalInfo
+              cautionFlags={it.cautionFlags}
+              expanded={expanded}
+              toggleExpansion={toggleExpansion}
+            />
+          </div>
+        )}
+
+        {!expanded && renderIconSection()}
+
+        <div className="card-bottom-cell vads-u-flex--1 vads-u-margin--0">
+          <div className="vads-u-padding--0 vads-u-margin-top--neg2 vads-u-margin-bottom--0p5">
+            <Checkbox
+              label="Compare"
+              checked={compareChecked}
+              onChange={handleCompareUpdate}
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+
+  return (
+    <div className="usa-grid vads-u-padding--0 vads-u-margin-bottom--4">
+      {renderProfilePageCard()}
+      {renderJumpLinks()}
+    </div>
+  );
+};
+
+ProfilePageHeader.propTypes = {
+  institution: PropTypes.object,
+  onGiBillLearnMore: PropTypes.func,
+  onAccreditationLearnMore: PropTypes.func,
+  onViewWarnings: PropTypes.func,
+};
+
+const mapStateToProps = state => ({
+  compare: state.compare,
+});
+
+const mapDispatchToProps = {
```

Comment:
Since this is being added this should be moved into the containers folder.
Also you might be able to now bring in `showModal` this way use it instead of passing in